test/test-trace-api.js

```diff
@@ -94,13 +101,14 @@ describe('Trace Interface', function() {
     
     before(function() {
       traceAPI.enable_(agent);
-      common.init(traceAPI);
-      common.avoidTraceWriterAuth(traceAPI);
+      traceAPI.private_().traceWriter.request = request;
```

Comment:
Yup, I think it would be good to expose this functionality from `common.js` even if it is only used here. Making sure `private_()` is only accessed through `common.js` is important to simplify future refactors that touch agent internals.